charts/eks-anywhere-packages/templates/cronjob.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  namespace: {{ .Values.namespace }}
+  name: cron-ecr-renew
+spec:
+  schedule: "0 */6 * * *"
+  successfulJobsHistoryLimit: 3
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: OnFailure
+          serviceAccountName: {{ include "eks-anywhere-packages.serviceAccountName" . }}
+          containers:
+            - name: {{.Values.cronjob.name}}
+              image: {{.Values.sourceRegistry}}{{ template "template.image" .Values.cronjob }}
+              imagePullPolicy: {{ .Values.imagePullPolicy }}
+              env:
+                - name: TARGET_NAMESPACE
+                  valueFrom:
+                    configMapKeyRef:
+                      name: {{.Values.nsconfigmap.name}}
+                      key: NAMESPACES
+                - name: DOCKER_SECRET_NAME
```

I don't think DOCKER in the name is great, but I'm not entirely sure how this is used.
```suggestion
                - name:ECR_TOKEN_SECRET_NAME
```

---

I am fine with this change reason I call it docker, as the secret type is kubernetes.io/dockerconfigjson even though its not really a docker specific type.